app/src/main/res/values/strings.xml

```diff
@@ -134,4 +134,5 @@
     <string name="permission_message">%s needs access to external storage. Please accept permission and try again.</string>
     <string name="toast_delete">Please delete file and try again</string>
     <string name="restore_database_message">Restore Data from Documents folder. Open folder \'Documents\' and select Cirrus.zip if needed!</string>
+    <string name="settings_show_hourly_uv_index">Show hourly UV index</string>
```

Comment:
How do translations work here? 
Via an external system or can I add a German translation?

Reply:
I would add translation for German and a few major languages. Others usually come with PRs